Readme.md

```diff
@@ -3,81 +3,85 @@
 
 # Pinpoint-Docker for Pinpoint
 
-Official git repository of Dockerized components of the [Pinpoint Application Monitoring](http://naver.github.io/pinpoint/).  
+Official git repository of dockerized components of the [Pinpoint Application Monitoring](http://naver.github.io/pinpoint/).  
 Installing Pinpoint with these docker files will take approximately 10min. to check out the features of pinpoint.
 
 ## What is Pinpoint
 
-[Pinpoint](https://github.com/naver/pinpoint), is the world's leading open-source application monitoring solution - trusted by millions of users around the globe.
-It supports and helps you understand your application in a glance and allow you to build world-class, high-quality software.
+[Pinpoint](https://github.com/naver/pinpoint), is the world's leading open-source application monitoring solution - trusted by millions of users around the world.
+It supports and helps you to understand your applications in a glance and allows you to build world-class, high-quality software.
```

Comment:
no need to include `to`, since it's optional.
Beside the `to`, you've got point